indexer/feature.cpp

```diff
@@ -375,6 +376,15 @@ void FeatureType::ParseMetadata() const
   m_metadataParsed = true;
 }
 
+void FeatureType::ParseAltitude() const
+{
+  if (m_altitudeParsed)
+    return;
+
+  m_pLoader->ParseAltitude();
```

Comment:
Please, do not handle altitudes here, in FeatureType. Move this code out of FeatureType, into individual unit.